app/models/manageiq/providers/openstack/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -144,6 +146,13 @@ def add_orchestration_stack_ancestry
     end
   end
 
+  def add_snapshots
```

Comment:
We have this collection in the base InfraManager [ref](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/infra_manager.rb#L51-L56) so we can just move this to shared [ref](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/shared.rb) so you don't have to re-define it.

Trying to get all of the provider persisters as close to core as possible

Reply:
I should say I'm good with this for now just an opportunity to clean things up in a follow-up pr